README.md

```diff
@@ -6,21 +6,72 @@
 
 This is an AWS IoT Thing simulator for nRF91. This project combines the [device-simulator](https://github.com/nRFCloud/device-simulator) and [dfu-device-simulator](https://github.com/nRFCloud/dfu-device-simulator) projects. It omits the legacy pairing mechanism and uses the Device API for creating JITP certs and associating a newly provisioned device with your tenant.
 
-### Getting Started
+## Getting Started
 ```sh
 # install deps
-npm i
+npm ci
+
+# create cache dir (for certs)
+mkdir cache
 
 # compile to js
-npx tsc
+npm run build
 
 # install jq
 https://stedolan.github.io/jq/download/
 ```
 
-### Commands
-See [simulator.ts](src/simulator.ts) for the options. Most of these are set with environment variables.
+## Options
+These are the options. Most of them are set with environment variables.
+
+```
+  -c, --certs-response <certsResponse>               JSON returned by call to the Device API: POST /devices/{deviceid}/certificates (default: "")
+  -e, --endpoint <endpoint>                          AWS IoT MQTT endpoint (default: "")
+  -d, --device-id <deviceId>                         ID of the device (default: "rand")
+  -o, --device-ownership-code <deviceOwnershipCode>  PIN/ownership code of the device (default: "123456")
+  -m, --mqtt-messages-prefix <mqttMessagesPrefix>    The prefix for the MQTT for this tenant for sending and receiving device messages (default: "")
+  -s, --services <services>                          Comma-delimited list of services to enable. Any of: [gps,acc,temp,device]
+  -a, --app-fw-version <appFwVersion>                Version of the app firmware (default: 1)
+  -k, --api-key <apiKey>                             API key for nRF Cloud (default: "")
+  -h, --api-host <apiHost>                           API host for nRF Cloud (default: "https://api.dev.nrfcloud.com")
+  -l, --link                                         Whether or not to automatically link device to account (default: false)
+  -v, --verbose                                      Verbose
+  -h, --help                                         Output usage information
+```
+
+Use `npx nrfsim --help` to see the most recent list of options.
```

Comment:
Regardless, I think we should keep the npx bin name the same as the project, which is more standard, e.g., instead of `nrfsim` use `@nrfcloud/device-simulator-v2`.

Reply:
That's so wordy though. But convention is convention... I'll update it.

Reply:
> Ah, probably b/c this is not yet published to npm.

Yeah, you can use the `npm link` to do testing.

Reply:
Adding the org name to the bin alias makes it unrunnable on install:

```bash
# this doesn't work
npx @nrfcloud/device-simulator-v2 -k <api key>

# this DOES work
npx device-simulator-vs -k <api key>
```

I believe this has to do with the way npm does it's alias linking on install. Adding any kind of slash into the path seems to make it create a directory into the `node_modules/.bin` directory. 

I'm okay with `device-simulator-v2` if you are...